packages-exp/auth-exp/test/helpers/integration/with_test_instance.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { initializeApp } from '@firebase/app-exp';
+import { initializeAuth } from '@firebase/auth-exp/index.browser';
```

Comment:
Is this the import path that customers will use? Just curious - it seems like it might take some time getting used to.

Reply:
@avolkovi has a better idea about this but I think so :/ The node package is the default index